multinet/uploaders/newick.py

```diff
@@ -82,14 +80,19 @@ def upload(workspace: str, table: str) -> Any:
     validate_newick(tree)
 
     space = db.db(workspace)
-    edgetable_name = "%s_edges" % table
-    nodetable_name = "%s_nodes" % table
+    if space.has_graph(graph):
+        raise AlreadyExists("graph", graph)
+
+    edgetable_name = "%s_edges" % graph
+    nodetable_name = "%s_nodes" % graph
```

Comment:
Use an f-string here.

Reply:
There's actually a bunch of places where the old style string formatting is being used, so it might be better to open a separate PR that addresses this throughout the application

Reply:
I'm a bit nervous to do it that way, because if we introduce bugs while changing them all out wholesale, those will be harder to catch than if we do so on a case-by-case basis as we run across these instances.

But go ahead and open that issue, and I'll recommend we change this instance now since it's in front of us.

Reply:
I see what you mean. I'll change this.

Reply:
I can only see 4 instances, all in the newick.py uploader. I may have missed some though